metrics-operator/config/crd/patches/cainjection_in_analyses.yaml

```diff
@@ -0,0 +1,7 @@
+# The following patch adds a directive for certmanager to inject CA into the CRD
```

Comment:
let's remove this file, as we also didn't keep that for the other CRD's

Reply:
yes you are right, done!